### PR TITLE
Add Box widget schema for redwood layout

### PR DESCRIPTION
**Pull request by underscoretang**

### CODE REVIEW

**Comment by JakeWharton** [comment]
LGTM so far

**Comment by underscoretang** [comment]
Current dependencies on/for this PR:
* trunk
  * **PR #1521** <a href="https://app.graphite.dev/github/pr/cashapp/redwood/1521" target="_blank">Graphite</a>  👈
    * **PR #1523** <a href="https://app.graphite.dev/github/pr/cashapp/redwood/1523" target="_blank">Graphite</a> 
      * **PR #1543** <a href="https://app.graphite.dev/github/pr/cashapp/redwood/1543" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/cashapp/redwood/1521?utm_source=stack-comment).

File: redwood-layout-schema/src/main/kotlin/app/cash/redwood/layout/widgets.kt [hunk 1]
```diff
@@ -169,3 +169,77 @@ public data class Spacer(
...
+   * Sets whether the box's height will match its tallest child ([Constraint.Wrap]) or match the
+   * height of its parent ([Constraint.Fill]).
+   */
+  @Property(4)
+  @Default("Constraint.Wrap")
+  val height: Constraint,
+
+  @Property(5)
+  @Default("Dp(0.0)")
+  val minHeight: Dp,
```
Comment (underscoretang) [inline redwood-layout-schema/src/main/kotlin/app/cash/redwood/layout/widgets.kt]: Do folks think that these would be useful?

Comment (jingwei99) [reply redwood-layout-schema/src/main/kotlin/app/cash/redwood/layout/widgets.kt]: I think it's fine to be here, as long as it is a no-op when not assigned

**Review by jingwei99 — APPROVED** [review]

File: redwood-layout-schema/src/main/kotlin/app/cash/redwood/layout/widgets.kt (hunk 1 above)
Comment (JakeWharton) [reply redwood-layout-schema/src/main/kotlin/app/cash/redwood/layout/widgets.kt]: I would not add any new properties that are not fundamentally intrinsic to the display of Box. Things like minHeight/minWidth are equally as applicable to Row or Column, and we should discuss their inclusion separately so as to not distract from getting Box landed.

Comment (underscoretang) [reply redwood-layout-schema/src/main/kotlin/app/cash/redwood/layout/widgets.kt]: cc: @swankjesse going to remove these

**Pull request merged**